optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -968,6 +986,8 @@ def __init__(
         self.lgbm_kwargs["show_stdv"] = show_stdv
         self.lgbm_kwargs["seed"] = seed
         self.lgbm_kwargs["fpreproc"] = fpreproc
+        if return_cvbooster is not None:
+            self.lgbm_kwargs["return_cvbooster"] = return_cvbooster
```

Comment:
Could you check the version of `lightgbm` to notify this feature is only for `lightgbm>=3.0.0`?
If users use `lightgbm<3.0.0`, they will get the following error and I think it is not user-friendly.

```console
  File "/Users/yanase/pfn/code/optuna/optuna/study.py", line 778, in _run_trial
    result = func(trial)
  File "/Users/yanase/pfn/code/optuna/optuna/integration/_lightgbm_tuner/optimize.py", line 305, in __call__
    cv_results = lgb.cv(self.lgbm_params, self.train_set, **self.lgbm_kwargs)
TypeError: cv() got an unexpected keyword argument 'return_cvbooster'
```

This is just and example. Please feel free to update it. Please note that the following code requires `packaging.version`.
```suggestion
        if return_cvbooster is not None:
            if version.parse(lgb.__version__) < version.parse("3.0.0"):
                raise ValueError("return_cvbooster requires lightgbm>=3.0.0.")
            self.lgbm_kwargs["return_cvbooster"] = return_cvbooster
```